src/skip-to-content/skip-to-content.scss

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-skip-to-content {
+  @include iui-blur($opacity: 3);
```

Comment:
Minor and not connected to this PR, but we should remove `backdrop-filter` from themed mixin in `iui-blur`
![image](https://user-images.githubusercontent.com/81580355/153560661-d9a74d88-a786-43b6-b8ea-75ab15446127.png)

Reply:
@elephantcatdog I think Vykintas meant just move it out of the mixin, not completely remove it 😄

Like this:
```scss
@include themed() {
  background-color: rgba(0, 0, 0, t(iui-opacity-#{$opacity}));
}
backdrop-filter: $iui-blur-filter;
```

Although, now that I think about, if `backdrop-filter` is supported, then you don't need the IE fallback and can just use CSS variables directly:
```scss
background-color: rgba(0 0 0 / var(--iui-opacity-#{$opacity}));
backdrop-filter: $iui-blur-filter;
```

Reply:
Oh, whoops, thanks for clarifying.

Reply:
"backdrop-filter" does not seem to be supported on IE11, [according to caniuse](https://caniuse.com/css-backdrop-filter).

Reply:
yes, that's why we can use CSS variables. The code is wrapped inside `@supports (backdrop-filter: $iui-blur-filter)` so it will not be used in IE.

Reply:
Fixed

Reply:
this is now wrong syntax, `t` should only be used inside `themed` mixin.

my original comment above had the right syntax (use `var` directly)